geode-serialization/src/main/java/org/apache/geode/internal/serialization/DataSerializableFixedID.java

```diff
@@ -225,7 +225,10 @@
 
   byte REMOVE_CACHESERVER_PROFILE_UPDATE = -19;
 
-  // IDs -18 through -10 unused
+  byte REGION_REDUNDANCY_STATUS = -18;
+  byte RESTORE_REDUNDANCY_RESULTS = -17;
```

Comment:
Nitpick - but maybe we should save these small message ids for more frequent messages?

Reply:
Sounds sensible